Document the new Maintenance Windows feature (#3253)
elastic/security-docs: docs/detections/rules-ui-manage.asciidoc

** *Add rule actions*: Add <<rule-notifications,notification actions>> on all selected rules.
+
NOTE: The action frequency you select applies to all actions (both new and existing) on all selected rules. If you don't want to change the frequency of existing actions, update the rules separately.
+

> NOTE: Rule actions that you add during a {ref}/maintenance-windows.html[maintenance window] won't run. They'll resume running after the maintenance window ends.

Reviewer: Will `{kibana-ref}` fix the cross-docs link? Also, I wonder if this should be a little more general -- I know it's in the context of creating a rule, adding an action, but it might be more accurate to say that _any_ rule actions won't run during a window, not just the ones that you add _during_ a window.
```suggestion
NOTE: Rule actions won't run during a {kibana-ref}/maintenance-windows.html[maintenance window]. They'll resume running after the maintenance window ends.
```

Author: Good point and the ref update should fix the angery link.
